### PR TITLE
Leaders component updates

### PR DESCRIPTION
**Pull request by manue1**
According to https://trello.com/c/gzFnUWWK/95-improve-city-pages

### CODE REVIEW

File: shared/components/common/leaders/leaders.component.js [hunk 1]
```diff
@@ -25,77 +27,32 @@ class LeadersComponent extends Panel {
...
-      leaders.props.updateUI(ui);
-      $(window).off('scroll', leaders.detectScroll());
-    }
   }
 
   componentWillUnmount(){
     $(window).off('scroll', this.detectScroll());
     $(document).off('click.hideLocations');
+    $(document).off('click.hideHouseholdList');
   }
```
Comment (erichulburd) [inline shared/components/common/leaders/leaders.component.js:50]: 👌

**Review by erichulburd — CHANGES_REQUESTED** [review]
Can you:

1) resolve conflicts, 
2) include a smoke test procedure I can run through when reviewing?

**Comment by manue1** [comment]
Smoke test procedure:
1. Pull this branch once again (I rebased from master to resolve conflicts)
2. Run [API](https://github.com/arbolista-dev/cc-user-api/pull/8) from features/leaders branch
3. Go to 'Your Footprint' component to make sure retrieving works and displays error for empty list of leaders
4. Sign up new user
5. Go to 'Get Started' and update any value (Up until now user answers only get sent to API once the defaults have been changed, not sure if you would rather send the default values to API on sign up and then once they get changed update them?)
6. Go back to 'Your Footprint' to find your user in list.
7. Test household size filter by selecting value other than the one chosen for user
8. In order to test location filter you'd need to sign up an additional user with different location

**Review by erichulburd — APPROVED** [review]

**Comment by erichulburd** [comment]
Thanks for the smoke test - this is very useful so I know what I'm looking for. Let's do this on all new PRs

**Pull request merged**